src/docker-images/RepairManagerAgent/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:18.04
+MAINTAINER Deborah Sandoval <Deborah.Sandoval@microsoft.com>
+
+RUN apt-get update && apt-get --no-install-recommends install -y \
+    python3.5 \
+    python3-pip \
+    systemd-sysv
+
+RUN pip3 install wheel
+RUN pip3 install setuptools
+RUN pip3 install requests
+RUN pip3 install pyyaml
```

Comment:
Make it one line. Multiple lines will generate multiple image layers which will make pulling unncessary slow. For ref https://hackernoon.com/tips-to-reduce-docker-image-sizes-876095da3b34